styles.css

```diff
@@ -491,178 +549,199 @@ hr {
- }
-
-.logo {
-  margin-left: 2em;
- }
-
-.mobile-icons {
-  display: none;
- }
-
-.home-icons {
-  display: flex;
- }
-
-.P-Text {
-  align-content: center;
-  width: 63%;
- }
-
-.P-Text h1 {
-  font-size: 48px;
- }
-
-.P-Text p {
-  font-size: 20px;
- }
-
-.page-2-description {
-  width: 100%;
- }
-
-.project1 {
-  flex-direction: row;
-  column-gap: 1rem;
-  width: 90%;
- }
-
-.project2 {
-  flex-direction: row-reverse;
-  column-gap: 1rem;
-  width: 90%;
- }
-
-.project3 {
-  flex-direction: row;
-  column-gap: 1rem;
-  width: 90%;
- }
-
-.project4 {
-  flex-direction: row-reverse;
-  column-gap: 1rem;
-  width: 90%;
- }
-
-.web-icons{
-  width: 394px;
- }
-
-.pager-head {
-  font-size: 38px;
-  line-height: 52px;
- }
-
-.program-used {
-  padding: 20px;
- }
-
-.ul-details {
-  font-size: 12px;
- }
-
-.p-details {
-  font-size: 18px;
- }
-
-.p-ul-details {
-  width: 80%;
-  font-size: 16px;
-  line-height: 24px;
- }
-
-.about {
-  flex-direction: row;
-  height: 628px;
-  justify-content: center;
-  align-items: center;
-  padding: 0 8em 0 8em;
-  column-gap: 2em;
- }
-
-.user-tool {
-  width: 60%;
- }
-
-.user-tool-d {
-  width: 60%;
-  padding-top: 20px;
-  margin-bottom: 20px;
- }
-
-.me {
-  width: 70%;
- }
-
-.me p {
-  width: 447px;
-  padding-bottom: 15px;
- }
-
-.me h2 {
-  padding-bottom: 15px;
- }
-
-.tool-list {
-  display: flex;
-  flex-direction: row;
-  width: 100%;
-  padding-top: 38px;
- }
-
-.tool-list > li {
-  height: 122px;
-  flex-direction: column;
-  align-items: flex-start;
-  justify-content: center;
-  margin: 0 1em 1em 0;
-  width: 120px;
- }
-
-.skills {
-  padding-top: 20px;
- }
-
-.contact-form p {
-  width: 47%;
- }
-
-.respo-icons {
-  display: none;
- }
-
-.frameworks {
-  padding-top: 20px;
- }
-
-.ul-form {
-  list-style: none;
- }
-
-#name_form {
-  padding-left: 20px;
- }
-
-#email_form {
-  padding-left: 20px;
- }
-
-#message {
-  padding-left: 20px;
- }
-
-.main-icons {
-  display: block;
-  width: 55%;
-  height: auto;
-  margin: 10px 0 10px 10px;
- }
+  .container1 {
+    height: 916px;
+  }
+
+  .img {
+    display: none;
+  }
+
+  nav {
+    height: 72px;
+  }
+
+  .logo {
+    margin-left: 2em;
+  }
+
+  .mobile-icons {
+    display: none;
+  }
+
+  .home-icons {
+    display: flex;
+  }
+
+  .P-Text {
+    align-content: center;
+    width: 63%;
+  }
+
+  .P-Text h1 {
+    font-size: 48px;
+  }
+
+  .P-Text p {
+    font-size: 20px;
+  }
+
+  .page-2-description {
+    width: 100%;
+  }
+
+  .project1 {
+    position: absolute;
+    margin-top: -49px;
+    flex-direction: row;
+    column-gap: 1rem;
+    width: 90%;
+  }
+
+  .project2 {
+    position: absolute;
+    flex-direction: row-reverse;
+    column-gap: 1rem;
+    width: 90%;
+  }
+
+  .project3 {
+    position: absolute;
+    margin-top: 49px;
+    flex-direction: row;
+    column-gap: 1rem;
+    width: 90%;
+  }
+
+  .project4 {
+    margin-top: 99px;
+    flex-direction: row-reverse;
+    column-gap: 1rem;
+    width: 90%;
```

Comment:
- [ ] As requested by the previous code reviewer, the project cards are supposed to have the same dimensions, including height and width, however, I see that some of the cards have different dimensions as shown in the image below. :point_down: 

- [ ] Also, from the Figma design, the project card is divided into two, half of it contains the project's picture, and the other half contains the project's description. however, while working on the changes requested by the previous code reviewer, your project images are not showing so make sure that you fix them. :smiley: 

- [ ] It would be better if you, adjust the margin left and right of your project design to match the Figma design. :blush: 

  Kindly check and fix these issues to match the actual design and produce a more professional project. :pray: 
  
  ### YOUR DESIGN
  ![Screenshot from 2023-02-25 22-06-30](https://user-images.githubusercontent.com/92303891/221381718-521e39cc-a21d-4a3b-b785-1e56c404d3ae.png)![Screenshot from 2023-02-25 22-07-03](https://user-images.githubusercontent.com/92303891/221381717-69ae6b71-3ce8-471b-b60d-5845ac877f02.png)![Screenshot from 2023-02-25 22-07-15](https://user-images.githubusercontent.com/92303891/221381714-4c130c95-b937-4cbb-b48c-a7648fca10ab.png)
  
  ### Figma Design
  ![image](https://user-images.githubusercontent.com/92303891/221382035-9a3cfbaf-2875-4997-b51f-18ee16b4ab32.png)